fastly/computeacl/v1/api_list_acls.go

```diff
@@ -0,0 +1,25 @@
+package v1
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/fastly/go-fastly/v9/fastly"
+)
+
+// ListACLs retrieves all compute ACLs.
```

Comment:
Interestingly, the API doesn't provide pagination for the ACLs. 
It is not a problem for this PR to solve, but I thought I'd at least highlight it.